src/domain/community/pendingMembership/PendingMemberships.tsx

```
@@ -72,8 +75,17 @@ type InvitationHydratorProps = {
     }
 );
 
-export const getChildJourneyTypeName = ({ level }: { level: JourneyLevel }): JourneyTypeName =>
-  ['space', 'subspace', 'subsubspace'][level] as JourneyTypeName;
+export const getChildJourneyTypeName = ({ level }: { level: SpaceLevel }): JourneyTypeName => {
+  switch (level) {
+    case SpaceLevel.Challenge:
+      return 'subspace' as JourneyTypeName;
+    case SpaceLevel.Opportunity:
+      return 'subsubspace' as JourneyTypeName;
+    case SpaceLevel.Space:
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)